### PR TITLE
110 - Review Page

### PR DESCRIPTION
**Pull request by vincentchiang1**
Only works with part 1 currently, parts 2 + 3 are commented out so i can uncomment it and test it after those are merged in! Mainly just replaced the mock input with real user application, added auto save and fixed T&C links not opening

image
image

image

### CODE REVIEW

File: src/components/ApplicationForm/ReviewCards.js [hunk 1]
```diff
@@ -88,28 +88,44 @@ const ethnicityOptions = {
   preferNot: 'Prefer not to say',
 }
 
+const eventOptions = {
+  option1: 'Local Hack Day / Hack Camp',
```
Comment (jackyzha0) [inline src/components/ApplicationForm/ReviewCards.js:92]: ONE WORD VINCENT ONE WORD stylization wall of shame u go

Comment (vincentchiang1) [reply src/components/ApplicationForm/ReviewCards.js:92]: oof i will fix thank you! i copied that from somewhere else smh

**Comment by anneguo3** [comment]
@vincentchiang1 I just merged part 2, so feel free to pull it and add it to your branch!

**Comment by anneguo3** [comment]
just merged in part 3 too, so probably do it all at once LOL

**Comment by vincentchiang1** [comment]
aighttt from what i can tell, part 2 is working with Anne's changes, just had to add the regex matching so the path doesn't get displayed on the review screen! i uncommented part 3 too, but it's empty rn obv since the values aren't uploaded to firebase, that should also be resolved once @kevin-zou 's PR is merged!

**Comment by panjenny0** [comment]
> aighttt from what i can tell, part 2 is working with Anne's changes, just had to add the regex matching so the path doesn't get displayed on the review screen (since firebase stores the path)! i uncommented part 3 too, but it's empty rn obv since the values aren't uploaded to firebase, that should also be resolved once @kevin-zou 's PR is merged!

@kevin-zou's PR is merged! the entire application should be good to go

File: src/containers/Application/Review.js [hunk 2]
```diff
@@ -2,70 +2,46 @@ import React from 'react'
...
+        handleEdit={handleEdit}
+        onChange={updateTermsAndConditions}
+      />
       <NavigationButtons
         firstButtonText="Back"
         firstButtonOnClick={() => handleNavigation('/application/part-3')}
         secondButtonText="Submit"
         secondButtonOnClick={() => handleNavigation('/application/confirmation')}
-        autosaveTime="4:20am" // TODO: replace with application.submission.lastUpdated.toDate().toString()
+        autosaveTime={application.submission.lastUpdated.toDate().toString()} // TODO: replace with application.submission.lastUpdated.toDate().toString()
```
Comment (kevin-zou) [inline src/containers/Application/Review.js:44]: Nit: can remove comment here I think

**Review by kevin-zou — APPROVED** [review]
It's lit 🔥

File: src/containers/Application/Review.js (hunk 2 above)
Comment (vincentchiang1) [reply src/containers/Application/Review.js:44]: i'd remove all the comments in the world for you babe

Comment (vincentchiang1) [reply src/containers/Application/Review.js:44]: good catch ty!

**Review by panjenny0 — APPROVED** [review]
Looks good! I think this is good to merge

**Pull request merged**